truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(isEmployed, isOnVacation));
+
+// 2
+function closeCompare(numberA, numberB, numbersMargin){
```

Comment:
`numberA` and `numberB` are not great names, try `firstNumber` and `secondNumber` instead